Setup/UpgradeData.php

```diff
@@ -182,6 +200,51 @@ public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface
             );
         }
 
+        /**
+         * Create Importer of Record Override Option
+         */
+
+        /** @var CustomerSetup $customerSetup */
+
+
+        if (version_compare($context->getVersion(), '2.0.3', '<')) {
+
+            $customerSetup = $this->customerSetupFactory->create(['setup' => $setup]);
+            $customerEntity = $customerSetup->getEavConfig()->getEntityType('customer');
```

Comment:
You have a bunch of unhandled exceptions in this method. If it doesn't make sense to handle them, add them to the PHPDoc